### DISCUSSION TITLE
How to show mails from same user in one chat box?

### DISCUSSION THREAD

**Discussion by gulsenkeskin** [chatwoot/chatwoot#4484] [General]
While the e-mails I sent to a user in Chatwoot appear in the same chat box, the e-mails I received appear in different chat boxes. Is there no way to show them in single chat box?

**Comment by sojan-official** [comment]
@gulsenkeskin  there was a bug in our previous version which caused this. ref: https://github.com/chatwoot/chatwoot/issues/3868
It should be fixed now

**Reply by gulsenkeskin** [reply]
Thank you.
